ofctrl/dial_windows.go

```diff
@@ -0,0 +1,11 @@
+package ofctrl
+
+import (
+	"github.com/Microsoft/go-winio"
+	"net"
+)
+
+// Connect to named pipe
+func DialUDSOrNamedPipe(address string) (net.Conn, error) {
+	return winio.DialPipe(address, nil)
+}
```

Comment:
Please reformat the code. There is a reminder of "No new line at the end of file" currently in all your new added files.

Reply:
Thanks Wenying. I have update the commit to append new line at end of source files.